app/views/conference_registrations/show.html.haml

```diff
@@ -116,8 +116,12 @@
       -if @registration
         .btn-group-vertical.pull-right
           = link_to 'Edit your Registration', edit_conference_conference_registration_path(@conference.short_title), class: 'btn btn-success', disabled: @conference.end_date < Date.today
-          = link_to 'Unregister', conference_conference_registration_path(@conference.short_title),
-          method: :delete, class: 'btn btn-danger btn-xs', confirm: 'Are you sure you want to unregister?', disabled: @conference.end_date < Date.today
+          - if @tickets.any?
+            = link_to 'Unregister', conference_conference_registration_path(@conference.short_title), method: :delete, class: 'btn btn-danger btn-xs',
+              data: { confirm: "Your ticket purchases won't be deleted. Are you sure you want to unregister?" }, disabled: @conference.end_date < Date.today
```

Comment:
I don't think that deleting a ticket purchase makes much sense to an attendee. What we should be saying - and I assume that's what you meant to communicate here - is that there will be no refund.